app/models/financialdata/FinancialDataQueryParameters.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.financialdata
+
+
+import play.api.libs.json.{Format, Json}
+
+import java.time.LocalDate
+
+final case class FinancialDataQueryParameters(
+                                               fromDate:                   Option[LocalDate] = None,
```

Comment:
One to keep in mind, Tim said that the API will expect all of these fields as mandatory. Although I assume yous have looked at the API spec in Jira which says theyre optional. Ill confirm this.